gradio/components/audio.py

```diff
@@ -310,6 +310,10 @@ def postprocess(
 
     @staticmethod
     def _convert_to_adts(data: bytes):
+        if wasm_utils.IS_WASM:
```

Comment:
I would also raise if `needs_conversion` is `True` in `preprocess`. I am pretty sure that `pydub` is using ffmpeg under the hood in that case.